[2025-06-29T20:32:40.374+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: superstore_etl_workflow.cargar_fact_ventas manual__2025-06-29T20:04:38.635006+00:00 [queued]>
[2025-06-29T20:32:40.390+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: superstore_etl_workflow.cargar_fact_ventas manual__2025-06-29T20:04:38.635006+00:00 [queued]>
[2025-06-29T20:32:40.391+0000] {taskinstance.py:2170} INFO - Starting attempt 4 of 5
[2025-06-29T20:32:40.413+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): cargar_fact_ventas> on 2025-06-29 20:04:38.635006+00:00
[2025-06-29T20:32:40.420+0000] {standard_task_runner.py:60} INFO - Started process 859 to run task
[2025-06-29T20:32:40.425+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'superstore_etl_workflow', 'cargar_fact_ventas', 'manual__2025-06-29T20:04:38.635006+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/superstore_etl_dag.py', '--cfg-path', '/tmp/tmp6r_w0t13']
[2025-06-29T20:32:40.428+0000] {standard_task_runner.py:88} INFO - Job 33: Subtask cargar_fact_ventas
[2025-06-29T20:32:40.492+0000] {task_command.py:423} INFO - Running <TaskInstance: superstore_etl_workflow.cargar_fact_ventas manual__2025-06-29T20:04:38.635006+00:00 [running]> on host 85c12c2be069
[2025-06-29T20:32:40.597+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='tu_nombre' AIRFLOW_CTX_DAG_ID='superstore_etl_workflow' AIRFLOW_CTX_TASK_ID='cargar_fact_ventas' AIRFLOW_CTX_EXECUTION_DATE='2025-06-29T20:04:38.635006+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-29T20:04:38.635006+00:00'
[2025-06-29T20:32:40.600+0000] {logging_mixin.py:188} INFO - Iniciando ETL para la tabla de hechos de ventas...
[2025-06-29T20:32:40.601+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-29T20:32:40.611+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=superstore_etl_workflow, task_id=cargar_fact_ventas, execution_date=20250629T200438, start_date=20250629T203240, end_date=20250629T203240
[2025-06-29T20:32:40.638+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-29T20:32:40.660+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
